### PR TITLE
fix(root): updated accesible labels for anchor and left hand navs

### PR DESCRIPTION
**Pull request by GCHQ-Developer-741**
## Summary of the changes
Updated accessible labels for the left hand nav and anchor nav to be more descriptive regarding what a user is focused on, giving them more information

## Related issue
#610 

## Checklist
- [x] I have [manually accessibility tested](https://design.sis.gov.uk/accessibility/testing/manual-testing) any changes, if relevant.
- [x] I have raised this pull request against the [develop branch](https://github.com/mi6/ic-design-system/tree/develop)

### CODE REVIEW

File: src/components/AnchorNav/index.tsx [hunk 1]
```diff
@@ -96,9 +91,16 @@ const AnchorNav: React.FC<AnchorNavProps> = ({
...
 
+  const currentPageName = currentPage.substring(
+    currentPage.lastIndexOf("/") + 1
+  );
+
   return headings.length > 0 ? (
     <div className="side-nav">
-      <nav aria-label={`${section} section contents`} className="nav">
+      <nav
+        aria-label={`${currentPageName} ${activeId} page contents`}
```
Comment (GCHQ-Developer-465) [inline src/components/AnchorNav/index.tsx:101]: When testing, the screen reader does not pick this up. 

For instance here is the alert page, the screen reader should read out "alert introduction page contents" which is there when inspecting (as seen below).

Screenshot 2024-01-19 at 10 41 45

This is the output of the screen reader.
Screenshot 2024-01-19 at 10 37 35

I presume this is a screen reader bug but is worth looking into to? 
Thought to raise this. 

Great work though 👍

Comment (GCHQ-Developer-741) [reply src/components/AnchorNav/index.tsx:101]: Yeah the aria-label describes the nav area as a whole, whereas your screenshot appears to be reading out an individual nav item.

**Review by GCHQ-Developer-299 — DISMISSED** [review]
Tested with Voiceover in Chrome, both read very nicely now! 
(note to other devs: I found it easy to use VO Rotor Ctrl+Opt+U to see full list of landmarks and the improved labels)

**Review by GCHQ-Developer-465 — APPROVED** [review]
This is great work! I have further compared it with the current implementation and it has made a great difference!

**Review by GCHQ-Developer-465 — APPROVED** [review]

**Review by MI6-255 — APPROVED** [review]

**Pull request merged**